scripts/jenkins/ardana/gerrit/gerrit_handle_event.py

```diff
@@ -168,6 +168,8 @@ def handle_change_updated(change, dry_run=False):
                                     '' if direct else 'in',
                                     change.gerrit_url)
                           )
+            gerrit_review(ref_change, label='Code-Review', vote=0,
```

Comment:
I should've mentioned this in the Jira ticket: the Gerrit trigger ignores events triggered in the context of its own Gerrit user. So even if you post a comment here, it will ignore it. [Apparently](https://stackoverflow.com/questions/48351193/gerrit-trigger-is-not-triggering-jenkins-job-based-on-the-comment-posted-to-patc) this has something to do with the "build current patches only" feature, which we have enabled, so we should either disable it and use our own means of aborting previous jobs (which incidentally you also include in this same PR), or find a way around it, such as circumventing Gerrit and starting a Jenkins job ourselves.

Reply:
I asked @rsalevsky to temporarily disable the "build current patches only" option in our Gerrit Jenkins trigger configuration. Now we can test to see if this new approach works better.

Reply:
Okay, good to know. I'll look to do it via the jenkins api.

Reply:
> Even with those options disabled, the Gerrit Jenkins trigger is still ignoring its own comments...

I don't know if it was due to a Jenkins server update, or a Gerrit trigger plugin restart, but now this works. Which means we can go back to using this far simpler solution instead of interacting with the Jenkins API directly.